src/test/resources/playgroundConf.json

```diff
@@ -11,26 +11,40 @@
       "title": "Drive",
       "widgets": [
         {
-          "height": "auto",
+          "height": "3",
           "id": "testVarEditor",
           "kwargs": {},
+          "resize": false,
```

Comment:
This functionality seems to be broken because I can resize this widget.

Reply:
This works on my machine. If you get a chance can you check dev tools and make sure there is a `data-gs-no-resize` property?